### DISCUSSION TITLE
laravel vapor

### DISCUSSION THREAD

**Discussion by ganesha-ds** [opcodesio/log-viewer#322] [Q&A] [answered]
This package will work properly with Laravel vapor.

**Comment by arukompas** [comment, marked as answer]
hey @ganesha-ds , Laravel Vapor doesn't have a filesystem that's constant, and so there are no plans yet to support Laravel Vapor.

This package works best with a `file` log driver, where logs are stored in plain text files on the same filesystem as the Laravel application.
